client/state/selectors/is-jetpack-remote-install-complete.js

```diff
@@ -0,0 +1,18 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns success status of an attempted remote install
+ * of the jetpack plugin to the .org site at the given url.
+ *
+ * @param {Object} state - Global state tree
+ * @param {String} url - .org site URL
+ * @return {bool} - True if installation and activation was successful
+ */
+export default function isJetpackRemoteInstallComplete( state, url ) {
+	return !! get( state.jetpackRemoteInstall.isComplete, [ url ], false );
```

Comment:
I'm inclined to leverage the safety of pushing everything into the second argument.